githubaction/0.1.0/README.md

```diff
@@ -0,0 +1,257 @@
+# GitHub Actions via Keptn Webhooks
```

Comment:
Please change to: 

`# GitHub Actions Webhook Integration`

Reply:
fixed in https://github.com/keptn-sandbox/artifacthub/pull/34/commits/a0ab4afe44744d3ee98494d4f368c60a5f18bea4